EnglishTranslation-master/ETC.tsv

```diff
@@ -15878,7 +15878,7 @@ ETC_20151016_015879	There aren't useful components
 ETC_20151016_015880	Destroying Demon Totems
 ETC_20151016_015881	Retreat Order!
 ETC_20151016_015882	Blew the horn to order for retreat
-ETC_20151016_015883	Checking the Trace
+ETC_20151016_015883	Checking the Evidence
```

Comment:
Recently, I've been using the word `examine`/`examining` instead of the generic `check`. What do you say?